docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -137,447 +113,228 @@ If you are using an older version, please upgrade.
 
-            The settings on the *IPv4* tab will remain blank because we are not
-            connected yet. On the *Conn.* tab, check the box for *Connect automatically*.
-            This way you don't have to connect manually after you reboot.
+    13. Select *Change...*.
 
-            {% include screenshot.html source="/images/brickman/networking-connect-automatically-selected.png" %}
+        {% include screenshot.html source="/images/brickman/wired-status-disconnected-ipv4-empty.png" %}
 
-            Then select *Connect*. *State* should change to *Online*. Go back to
-            the *IPv4* tab. It will show the IP address of your EV3.
+    14. Then choose *Load Windows Defaults*.
 
-            {% include screenshot.html source="/images/brickman/networking-ipv4-tab-with-windows-defaults.png" %}
+        {% include screenshot.html source="/images/brickman/networking-load-windows-defaults.png" %}
 
-        30. Now we are going to connect to the EV3 using ssh. To do this, you need a
-            program called [PuTTY].
+    15. The settings on the *IPv4* screen will remain blank because we are not
+        connected yet. Go back to the previous screen and check the box for
+        *Connect automatically*. This way you don't have to connect manually
+        after you reboot. Then select *Connect*. *State* should change to
+        *Online* to indicate that your EV3 is connected to the Internet.
 
-            Start PuTTY. In the *PuTTY Configuration* window, type the IP address that
-            we just saw (`192.168.137.3`). Then click the *Open* button to connect.
+        {% include screenshot.html source="/images/brickman/wired-status-online-connect-selected.png" %}
 
-            {% include screenshot.html source="/images/WindowsRNDIS/WindowsRNDISPutty.png" %}
+    16. Now we are going to connect to the EV3 using ssh. To do this, you need a
+        program called [PuTTY]. Start PuTTY. In the *PuTTY Configuration* window,
+        type in `ev3dev`. Then click the *Open* button to connect.
 
-            The first time you connect, you'll get a warning about the new RSA fingerprint.
+        {% include screenshot.html source="/images/windows/10/putty-configuration-ev3dev.png" %}
 
-            {% include screenshot.html source="/images/WindowsRNDIS/WindowsRNDISPuttyFingerprint.png" %}
+        {% include icon.html type="info" %}
+        The first time you connect, you'll get a warning about the new fingerprint.
+        This is normal. Just click *Yes* to continue. You won't see this again
+        unless you re-flash your SD card.
+        {% include screenshot.html source="/images/windows/10/putty-security-alert.png" %}
+        {: class="alert alert-info"}
 
-            Once you're connected, type in the userid as `root` and the password as `r00tme` and you should be connected.
+        Once you're connected, type in the userid as `robot` and the password as
```

Comment:
> Once you are connected, type in your username (`robot`) and your password
   (`ev3dev` if you haven't changed it yet) and then you should be logged in.

better?